docs/site/debugger.md

```diff
@@ -179,3 +179,47 @@ Most importantly, make sure you have `cider/cider-nrepl` as a dependency, and `c
 ```sh
 clojure -Sdeps '{:deps {nrepl/nrepl {:mvn/version,"0.8.3"},cider/cider-nrepl {:mvn/version,"0.25.8"}}}' -m nrepl.cmdline --middleware "[cider.nrepl/cider-middleware]"
 ```
+
+## Passing options to the REPL JVM
+
+There are times when clojure debugging tools are not enough or not right for the job.
+This is usually true when use an (open source) Java library and you want to set some brake points in Java code.
+For those cases and others, you need to start the JVM in debug mode.
+And for that you nee
```

Comment:
I think this sentence got cut off somehow.

Reply:
thanks, removed.